### ISSUE TITLE
Unable to add accessibility on Menu Component

### ISSUE THREAD

**Issue by PriyankaGandhi2504** [instea/react-native-popup-menu#206]
I used MenuOption in my code for displaying a menu like a overlay on the screen and added accessibilityLabel and testID to automate the component. But on checking in Appium (1.19.1) unable to locate the menu view. On clicking menu from Appium it is taking background screen component.
Let me know in case anyone know any solution for this. 
Please find below attached screen for the design reference.
 
<img width="184" alt="Screenshot 2021-02-24 at 10 59 32 PM" src="https://user-images.githubusercontent.com/55586118/109117217-b52e9d80-7767-11eb-8a19-6cf2f48212ab.png">
